src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -40,3 +40,10 @@ CREATE TABLE "student" (
   last_name TEXT NOT NULL,
   assigned_case_manager_id UUID REFERENCES "user" (user_id) ON DELETE SET NULL
 );
+
+CREATE TABLE "para" (
```

Comment:
maybe we want to use the `user` table instead of introducing a new one?

Reply:
definitely! should be correctly updated in the most recent commit now

Reply:
@codetheweb ^